src/boot/efi/meson.build

```diff
@@ -218,18 +219,21 @@ if have_gnu_efi
 
         libgcc_file_name = run_command(efi_cc + ['-print-libgcc-file-name']).stdout().strip()
         systemd_boot_efi_name = 'systemd-boot@0@.efi'.format(EFI_MACHINE_TYPE_NAME)
+        stub_so_name = 'linux@0@.so.stub'.format(EFI_MACHINE_TYPE_NAME)
```

Comment:
why call this "so"? Shouldn't this be called "elf"?

Reply:
previously, internally during build we would link .o into .so and objcopy into .efi. It is shared object, but it would be more clear to state that it is ELF and not EFI (pe/coff binary). I'll change it to elf.